tests/test_stats.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This module implements test coverage for the stats functions in stats.py.
+"""
+import numpy.random
+import scipy.stats
+
+import pytest
+from moztelemetry import stats
+
+
+def l2d(values):
+    # Convert a list of values to a histogram representation.
+    d = {}
+    for v in values:
+        d[v] = d.get(v, 0) + 1
+    return d
+
+
+# A normally distributed sample set.
+norm1 = list(numpy.random.normal(5, 3.25, 1000))
+norm2 = list(numpy.random.normal(5, 3.25, 1000))
+
+# A uniformly distributed sample set.
+uni1 = numpy.random.randint(1, 100, 1000)
+uni2 = numpy.random.randint(1, 100, 1000)
+
+# A skewed normal distribution.
+skew1 = list(scipy.stats.skewnorm.rvs(10, size=1000))
+skew2 = list(scipy.stats.skewnorm.rvs(10, size=1000))
+
+
+samples = {
+    'normalized': (norm1, norm2),
+    'uniform': (uni1, uni2),
+    'skewed': (skew1, skew2),
+}
+
+
+def test_rank():
+    assert stats._rank({1: 1}) == {1: 1.0}
+    assert stats._rank({1: 5, 2: 4, 3: 3, 4: 2, 5: 1}) == {
+        1: 3.0,
+        2: 7.5,
+        3: 11.0,
+        4: 13.5,
+        5: 15.0,
+    }
+
+
+def test_tie_correct():
+    assert stats._tie_correct({}) == 1.0
+    assert stats._tie_correct({1: 1}) == 1.0
+
+
+def test_ndtr():
+    # Test invalid values raise an error.
+    with pytest.raises(TypeError):
+        stats.ndtr(None)
+    with pytest.raises(ValueError):
+        stats.ndtr('a')
+
+    assert round(stats.ndtr(0), 6) == 0.5
+    assert round(stats.ndtr(1), 6) == 0.841345
+    assert round(stats.ndtr(2), 6) == 0.977250
+    assert round(stats.ndtr(3), 6) == 0.998650
+
+    assert round(stats.ndtr(0), 6) == round(scipy.special.ndtr(0), 6)
+    assert round(stats.ndtr(1), 6) == round(scipy.special.ndtr(1), 6)
+    assert round(stats.ndtr(1.5), 6) == round(scipy.special.ndtr(1.5), 6)
+    assert round(stats.ndtr(2), 6) == round(scipy.special.ndtr(2), 6)
+    assert round(stats.ndtr(3), 6) == round(scipy.special.ndtr(3), 6)
+
+
+def test_mann_whitney_u():
+    for sample in ('normalized', 'uniform', 'skewed'):
```

Comment:
These tests are close to what I was thinking, but all of these are comparing the same distribution. I would change one of the distributions (for each of: normalized, uniform, and skewed) to be different (different mean, different std dev, different number of samples, etc.), and in addition add tests that compare e.g. normalized to uniform, uniform to skewed, etc.